### DISCUSSION TITLE
Different way to get the chats

### DISCUSSION THREAD

**Discussion by ghost** [YuvrajRaghuvanshiS/WhatsApp-Key-Database-Extractor#64] [Ideas]
Hey,

so quick idea I wanted to run by you: 
Do you think an import like this could be possible for Signal as well?
https://telegram.org/blog/move-history 

Same question towards @jukefoxer , who wrote the import side of things.

**Comment by YuvrajRaghuvanshiS** [comment]
I can't say much, I'm not into Android development yet. And during my last conversation with Sam he seemed busy. He had an idea about Signal and he implemented it I think this now is too much to ask for haha.

**Reply by jukefoxer** [reply]
Hi Robert,
As the WhatsApp chat export option has a very different data format, I don't plan on extending the Signal import functionality.
Best,
Sam

**Comment by ghost** [comment]
Might be a bit unorthodox, but as Telegram is OpenSource as well, couldn't we just take a look how they do the import?
https://github.com/DrKLO/Telegram
